iib/web/models.py

```diff
@@ -1269,6 +1322,16 @@ def from_json(cls, kwargs, batch=None):
         request_kwargs['batch'] = batch
 
         request = cls(**request_kwargs)
+        db.session.add(request)
```

Comment:
Same as above

Reply:
Let's move this to `api_v1.py` and you will have to duplicate the validation here since you are not calling `cls._from_json` here since the Mixin class is not being inherited here.